pkg/connectivity/pod-to-pod.go

```diff
@@ -1,6 +1,7 @@
 package connectivity
 
 import (
+	"github.com/openservicemesh/osm-health/pkg/kubernetes/pod"
```

Comment:
The linter message is a bit ambiguous - but it will be addressed if this is in the last block of imports (last line) - separated with a new line from the other imports.

Reply:
AHHH these linter messages are being quite unhelpful. Thanks!